### PR TITLE
AddSpecificNumber algorithm added with tests

### PR DESCRIPTION
**Pull request by rodriguez-maria**

### CODE REVIEW

File: src/dev/AddSpecificNumber.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class AddSpecificNumber {
+	
+	public static int[] addNumber (int[] arr, int num){
+		if (arr == null || arr.length == 0)
+		{
+			return null;
+		}
+		
+		HashMap<Integer, Integer> map = new HashMap<Integer,Integer>();
+		int number;
```
Comment (rfaisal) [inline src/dev/AddSpecificNumber.java]: inside for

File: src/dev/AddSpecificNumber.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (rfaisal) [inline src/dev/AddSpecificNumber.java]: put the leet code link also

File: src/dev/AddSpecificNumber.java (hunk 1 above)
Comment (rfaisal) [inline src/dev/AddSpecificNumber.java]: rename it to something meaningful

Comment (rodriguez-maria) [reply src/dev/AddSpecificNumber.java]: done

File: src/dev/AddSpecificNumber.java (hunk 2 above)
Comment (rodriguez-maria) [reply src/dev/AddSpecificNumber.java]: done

File: src/dev/AddSpecificNumber.java (hunk 1 above)
Comment (rodriguez-maria) [reply src/dev/AddSpecificNumber.java]: done

**Pull request merged**